main/src/routes/webcamRouter.js

```diff
@@ -12,23 +12,27 @@ router.get('/', checkCredentials, function (req, res, next) {
 });
 
 router.post('/', [webcamLimiter, refreshCredentialsIfExpired], function (req, res, next) {
```

Comment:
Do you think we can use a middleware function to check the validity of the req body?

```suggestion
router.post('/', [webcamLimiter, refreshCredentialsIfExpired, checkWebcamPostBody], function (req, res, next) {
```

Reply:
Do you mean In order to check if it's null/undefined? Or to perform other checks over req body?

Reply:
So like currently I see in your code we check `req.body && req.body.dataURL`. I was thinking we can delegate checking the validity of the request body to a middleware function

Reply:
This is some kind of pattern I wanna follow for foodies as well to use middleware